freedomofpress/securedrop
# Add Czech to list of supported languages

## Status

Ready for review

## Description of Changes

Fixes #4882.

This adds Czech to the list of supported languages.

Also updates the `translation-test` script so it gets the list of supported locales from `i18n_tool.py` and will no longer have to be updated when we change that list.

## Testing

- Ensure that the Czech language code (`cs`) is included in the `SUPPORTED_LOCALES` variable in `securedrop/config.py`. You can edit the file to add it, or simply delete the file and run `make test-config`.
- Run `make dev` and verify that Czech is an option in the language menu:

![image](https://user-images.githubusercontent.com/47430598/66439445-ab36be00-e9fd-11e9-8cb1-87b83c896f2a.png)


## Deployment

This adds a new supported language. To offer it, sites will need to add it via `securedrop-admin sdconfig`, then run `securedrop-admin install`.

## Checklist

### If you made changes to the server application code:


### If you made non-trivial code changes:
